Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -67,7 +67,7 @@ class AuxiliaryProcessProxy
     WTF_MAKE_NONCOPYABLE(AuxiliaryProcessProxy);
 
 protected:
-    AuxiliaryProcessProxy(bool alwaysRunsAtBackgroundPriority = false, Seconds responsivenessTimeout = ResponsivenessTimer::defaultResponsivenessTimeout);
+    AuxiliaryProcessProxy(bool shouldTakeUIBackgroundAssertion, bool alwaysRunsAtBackgroundPriority = false, Seconds responsivenessTimeout = ResponsivenessTimer::defaultResponsivenessTimeout);
```

Comment:
Those 2 first boolean parameters seem a bit errors prone. Trivial for a call site to mix / reverse the 2. We should probably switch to enum classes.